### DISCUSSION TITLE
USB on Kali VM

### DISCUSSION THREAD

**Discussion by ExHo7** [cirruslabs/tart#834] [Q&A] [answered]
Hi, I'm currently using a Kali with Tart and it's great! However, I don't
have a USB connection. My official Apple adapter is not detected. The goal
is to connect an Alpha Wi-Fi adapter. Thank you for your help.

**Comment by edigaryev** [comment, marked as answer]
Unfortunately this is not possible at the moment, please see https://github.com/cirruslabs/tart/issues/139 for more details.

**Reply by ExHo7** [reply]
Thx for your quickly answer. I hope this feature will arrive soon.
